parsec/utils.py

```diff
@@ -42,28 +44,41 @@
 # - The backend compares the timestamp to its current time
 # The worse case scenario would be a slow client machine, a large
 # request, a slow network connection and a busy server. Even
-# in this scenario, a 30 seconds time difference is hardly
-# imaginable on a functionnning system and it still leaves 30
-# seconds to accept slighty desynchronized clocks.
+# in this scenario, a 10 seconds time difference is hardly
+# imaginable on a properly functionnning system.
 #
-# Still, this is an argument for making this comparison asymetrical: with no
+# This is an argument for making this comparison asymetrical: with no
 # clock drift between client and server, communication latency makes data
 # arriving to the backend always in the past. Hence we should be more
 # forgiving of data in the past than in the future !
 #
 # A more radical check would be to not accept more that 10 seconds
-# delay and 10 seconds shifting, yielding a -10/20 seconds window
+# delay and 10 seconds shifting, yielding a 10/20 seconds window
 # (10 seconds in advance or 20 seconds late). This would effectively
-# reduce the current -60/60 seconds time widown by a factor of 4.
-TIMESTAMP_MAX_DT = 60  # seconds
-
-
-def timestamps_in_the_ballpark(ts1: DateTime, ts2: DateTime, max_dt=TIMESTAMP_MAX_DT) -> bool:
+# reduce the current 50/70 seconds time window by a factor of 4.
+#
+# The ballpark client tolerance is the ratio applied to the offsets
+# while performing the ballpark checks. We use an arbitrary value of
+# 80% in order to make sure that a clock shift is caught during the
+# handshake instead of being caught by another API call later on.
```

Comment:
"A masterpiece of comment explanation" Telerama
![riad-sattouf-createur-des-ulysse-du-nouveau-telerama,M72721](https://user-images.githubusercontent.com/3187637/145557155-0209fcd4-7665-4830-90bb-056ede5f67a5.jpg)